sources/subs/Topic.subs.php

```diff
@@ -1708,16 +1701,21 @@ function selectMessages($topic, $start, $items_per_page, $messages = array(), $o
 		SELECT 
 			m.subject, COALESCE(mem.real_name, m.poster_name) AS real_name, 
 			m.poster_time, m.body, m.id_msg, m.smileys_enabled, m.id_member
-		FROM {db_prefix}messages AS m
-			LEFT JOIN {db_prefix}members AS mem ON (mem.id_member = m.id_member)
+		FROM (
+			SELECT 
+				m.id_msg 
+			FROM {db_prefix}messages AS m
 		WHERE m.id_topic = {int:current_topic}' . (empty($messages['before']) ? '' : '
 			AND m.id_msg < {int:msg_before}') . (empty($messages['after']) ? '' : '
 			AND m.id_msg > {int:msg_after}') . (empty($messages['excluded']) ? '' : '
 			AND m.id_msg NOT IN ({array_int:no_split_msgs})') . (empty($messages['included']) ? '' : '
 			AND m.id_msg IN ({array_int:split_msgs})') . (!$only_approved ? '' : '
 			AND approved = {int:is_approved}') . '
 		ORDER BY m.id_msg DESC
```

Comment:
Nitpicking it should be indented by 1 tab I think. :angel:

Reply:
Anything to help make sub selects a bit easier to read